internal/context/prompts.go

```diff
@@ -10,17 +10,30 @@ import (
 
 	"github.com/commitdev/zero/internal/config/globalconfig"
 	"github.com/commitdev/zero/internal/config/moduleconfig"
+	"github.com/commitdev/zero/pkg/credentials"
 	"github.com/commitdev/zero/pkg/util/exit"
 	"github.com/manifoldco/promptui"
 	"gopkg.in/yaml.v2"
 )
 
+// Constant to maintain prompt orders so users can have the same flow,
+// modules get downloaded asynchronously therefore its easier to just hardcode an order
+var AvailableVendorOrders = []string{"aws", "github", "circleci"}
+
+const AWS_PICK_PROFILE = "Existing AWS Profiles"
```

Comment:
Go convention is to use camel case, and most likely you want to start with lower case since this should be unexported.